src/core_ocean/analysis_members/mpas_ocn_mixed_layer_depths.F

```diff
@@ -230,11 +243,24 @@ subroutine ocn_compute_mixed_layer_depths(domain, timeLevel, err)!{{{
          call mpas_pool_get_array(diagnosticsPool, 'potentialDensity', &
                         potentialDensity)
          call mpas_pool_get_array(diagnosticsPool, 'pressure', pressure)
+         call mpas_pool_get_array(forcingPool, 'landIcePressure', landIcePressure)
+         call mpas_pool_get_array(forcingPool, 'landIceDraft', landIceDraft)
+         call mpas_pool_get_array(forcingPool, 'landIceMask', landIceMask)
          call mpas_pool_get_array(diagnosticsPool, 'zMid', zMid)
          call mpas_pool_get_array(diagnosticsPool, 'zTop', zTop)
          call mpas_pool_get_array(meshPool, 'latCell', latCell)
          call mpas_pool_get_array(meshPool, 'lonCell', lonCell)
 
+         pressureAdjustedForLandIce(:,:) = pressure(:,:)
+         !If landice cavity remove land ice pressure to search for ML depth
+         if ( associated(landIcePressure) ) THEN
+           do iCell = 1,nCellsSolve
+             do k = 1,maxLevelCell(iCell)
+                pressureAdjustedForLandIce(k,iCell) = pressureAdjustedForLandIce(k,iCell)   &
+                                                        - landIcePressure(iCell)
+             enddo
+           enddo
+         endif
 
          if(tThresholdFlag) then
             call mpas_pool_get_array(mixedLayerDepthsAMPool, 'tThreshMLD',tThresholdMLD)
```

Comment:
Strangely, we used tThreshMLD in `Registry_mixed_layer_depths.xml` and tThresh**old**MLD in the code. Same with dThreshMLD.  I think that is unnecessarily confusing, and easy to change here as *ThresholdMLD only appears in 8 lines, all in this module.